### DISCUSSION TITLE
The div html tag

### DISCUSSION THREAD

**Discussion by novetto** [reflex-dev/reflex#1720] [Q&A] [answered]
What component in reflex is equivalent to a div in html?

**Comment by SoufianeHammouda** [comment]


**Reply by masenf** [reply]
Reflex is a python-based full-stack web framework. It does not work with Ruby. Please refrain from answering questions with obviously incorrect information.

**Comment by masenf** [comment, marked as answer]
@novetto reflex provides standard HTML elements that you can use in your frontend code under the `reflex.el` namespace.

To make a div, you can use the following code

```python
import reflex as rx
...
def index():
    return rx.el.div("This is a standard HTML div tag.")
```
